docs/backup-recovery/README.md

```diff
@@ -0,0 +1,146 @@
+# Cluster upgrades - backup and recovery #
+
+## Background ##
+
+This feature creates a pre-upgrade backup and provides a procedure for rapid recovery of a SNO in the event of a failed upgrade. In case of an upgrade failure, this feature allows the SNO to be restored to a working state with the previous version of OCP without requiring a re-provision of the application(s).  
+
+## Create Recovery Partition ##
+
+> **WARNING**: It is highly recommended to create a recovery partition at install time if opted to use this feature.
+
+There are two methods related to partition creation is described here:
+
+### Preferred method with Siteconfig ###
+
+The recovery partition can be created at install time by defining a node level property called `diskPartition` in the SiteConfig. The configuration for the partition must be defined by `start` parameter which must indicate the end of the root device. Additionally, a spare disk can be used for recovery partition as well. A recovery partition of 50GB can be created by following the below example:
```

Comment:
Rephrase as you see it fit: 

The recovery partition can be created at install time by defining a node level property called `diskPartition` in the SiteConfig **CR**. The configuration for the partition must be defined by `start` parameter which must indicate, in mebibytes, the end of the /sysroot RHCOS partition and the start for your newly requested recovery partition. 

// the /sysroot partition (4th) will grow/expand until the start point of the requested recovery partition.

Additionally, a spare disk can be used for recovery partition as well and can be configured at provisioning time or at day 2 operation. A recovery partition of 50GB can be created by following the below example. Notice that the deviceName can be referenced by its path, label or id as well, e.g. /dev/disk/by-id/scsi-355cd2e404b7c23f3.

Reply:
lgtm